Cargo.toml

```diff
@@ -74,7 +75,7 @@ zeroize = { version = "1.5", default-features = false, features = [ "std" ], opt
 inx = { version = "1.0.0-beta", default-features = false, optional = true }
 tonic = { version = "0.8", default-features = false, optional = true }
 
-# Metrics
+# Opentelemetry
```

Comment:
Since you are doing a little `Cargo.toml` maintenance I am adding my 2 cents here ... 
I think we should call this section`Tracing`, and move `tracing-loki` into this section.
```
# Tracing
opentelemetry = ...
...
tracing-loki = ...
tracing-opentelemetry = ...
```

Also ... the `# Optional` section (see above) is not really distinct as most of our deps are optional. Since `chrono` is related to `tracing` as well, you can move it here, and then rebrand the `# Optional` section to `# Analytics`.

Reply:
I called it that because there is a feature `opentelemetry`. And the `Optional` section is for dependencies that are shared between features.